server/controller/root.py

```diff
@@ -106,6 +110,63 @@ def historical():
     }
 
 
+@get('/upload_logs')
+@kajiki_view('upload_logs')
+@authenticated()
+def upload_logs():
+    api_key = config.user_management_interface.get_api_key_for_user(request)
+    return {
+        'page': 'upload_logs',
+        'raw_logs': '',
+    }
+
+
+@post('/upload_logs')
+@kajiki_view('upload_logs')
+@authenticated()
+def upload_logs():
+    api_key = config.user_management_interface.get_api_key_for_user(request)
+
+    os_type = request.forms.get('os_type')
+    upload = request.files.get('upload')
```

Comment:
nitt: can you upload this as 'log_file' please. 'upload' is just a very generic name